config/filter.d/proftpd.conf

```diff
@@ -2,6 +2,9 @@
 #
 # Set "UseReverseDNS off" in proftpd.conf to avoid the need for DNS.
 # See: http://www.proftpd.org/docs/howto/DNS.html
+# When the default locale for your system is not en_US.UTF-8
+# be sure to add this to /etc/default/proftpd
```

Comment:
This is a Debian specific location, thus better to phrase so it is obvious

```

# These regular expressions devised for logs when locate is en_US.UTF-8.
# If your default locale is different, run proftpd under en_US.UTF-8 locale, e.g.
# on Debian-based systems add this to /etc/default/proftpd:
# export LC_TIME="en_US.UTF-8"
```
?